Maintenance test_010_create_mw_on_switch_should_move_evc has been flaky

This maintenance [`test_010_create_mw_on_switch_should_move_evc`](https://github.com/amlight/kytos-end-to-end-tests/blob/master/tests/test_e2e_15_maintenance.py#L86) test case has been a bit flaky recently:

```
=================================== FAILURES ===================================
_______ TestE2EMaintenance.test_010_create_mw_on_switch_should_move_evc ________
self = <tests.test_e2e_15_maintenance.TestE2EMaintenance object at 0x7fbe205eac70>
    def test_010_create_mw_on_switch_should_move_evc(self):
        """Tests the EVC behaviors during maintenance
        Test:
            /api/kytos/maintenance on POST
        """
    
        self.restart_and_create_circuit()
    
        # Sets up the maintenance window information
        mw_start_delay = 60
        mw_duration = 60
        start = datetime.now() + timedelta(seconds=mw_start_delay)
        end = start + timedelta(seconds=mw_duration)
    
        # Sets up the maintenance window data
        payload = {
            "description": "mw for test 010",
            "start": start.strftime(TIME_FMT),
            "end": end.strftime(TIME_FMT),
            "items": [
                "00:00:00:00:00:00:00:02"
            ]
        }
    
        # Creates a new maintenance window
        api_url = KYTOS_API + '/maintenance'
        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
        assert response.status_code == 201, response.text
        data = response.json()
        assert 'mw_id' in data
    
        # Waits for the MW to start
        time.sleep(mw_start_delay + 10)
    
        # Switch 1 and 3 should have 3 flows; Switch 2 should have only 1 flow.
        s1, s2, s3 = self.net.net.get('s1', 's2', 's3')
        flows_s1 = s1.dpctl('dump-flows')
        flows_s2 = s2.dpctl('dump-flows')
        flows_s3 = s3.dpctl('dump-flows')
        assert len(flows_s1.split('\r\n ')) == BASIC_FLOWS + 2
>       assert len(flows_s3.split('\r\n ')) == BASIC_FLOWS + 2
E       AssertionError: assert 6 == (3 + 2)
E        +  where 6 = len([' cookie=0x0, duration=90.459s, table=0, n_packets=0, n_bytes=0, priority=50000,dl_src=ee:ee:ee:ee:ee:01 actions=CONT...=92.360s, table=0, n_packets=56, n_bytes=2352, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=CONTROLLER:65535\r\n'])
E        +    where [' cookie=0x0, duration=90.459s, table=0, n_packets=0, n_bytes=0, priority=50000,dl_src=ee:ee:ee:ee:ee:01 actions=CONT...=92.360s, table=0, n_packets=56, n_bytes=2352, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=CONTROLLER:65535\r\n'] = <built-in method split of str object at 0x1d777a0>('\r\n ')
E        +      where <built-in method split of str object at 0x1d777a0> = ' cookie=0x0, duration=90.459s, table=0, n_packets=0, n_bytes=0, priority=50000,dl_src=ee:ee:ee:ee:ee:01 actions=CONTR...n=92.360s, table=0, n_packets=56, n_bytes=2352, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=CONTROLLER:65535\r\n'.split
```

However, a single execution sometimes doesn't fail (I don't have much statistics about it atm, since e2e on my computer is timeouting again, I'm trying to fix that). 

```
+ python3 -m pytest tests/ -k test_010_create_mw_on_switch_should_move_evc
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /builds/amlight/kytos-end-to-end-tester/kytos-end-to-end-tests
plugins: timeout-2.0.2
collected 199 items / 198 deselected / 1 selected
tests/test_e2e_15_maintenance.py .                                       [100%]
```

--- a/tests/test_e2e_15_maintenance.py
+++ b/tests/test_e2e_15_maintenance.py
@@ -115,7 +115,7 @@ class TestE2EMaintenance:
         assert 'mw_id' in data
 
         # Waits for the MW to start
-        time.sleep(mw_start_delay + 10)
+        time.sleep(mw_start_delay + mw_duration / 4)
 
         # Switch 1 and 3 should have 3 flows; Switch 2 should have only 1 flow.
         s1, s2, s3 = self.net.net.get('s1', 's2', 's3')

